src/Aspire.Hosting.Qdrant/QdrantContainerImageTags.cs

```diff
@@ -7,5 +7,5 @@ internal static class QdrantContainerImageTags
 {
     public const string Registry = "docker.io";
     public const string Image = "qdrant/qdrant";
-    public const string Tag = "v1.8.4";
+    public const string Tag = "v1.9.7";
```

Comment:
I see v1.11 now: https://hub.docker.com/layers/qdrant/qdrant/v1.11.0/images/sha256-22a2d455837380d5fa1a3455b87b4fe7af30aa4f4712f8a57027c61022113796?context=explore

Reply:
Thanks for spotting these, I'll do a pass again now that I realized that sorting was alphabetical to make sure we have latest for all

Reply:
Ok, fixed it.